parquet-tools/src/main/java/org/apache/parquet/tools/util/MetadataUtils.java

```diff
@@ -149,6 +146,13 @@ private static void showDetails(PrettyPrintWriter out, ColumnChunkMetaData meta,
     out.format(" FPO:%d", foff);
     out.format(" SZ:%d/%d/%.2f", tsize, usize, ratio);
     out.format(" VC:%d", count);
+
+    if (!meta.getStatistics().isEmpty()) {
+      out.format(" MIN:%s", meta.getStatistics().genericGetMin().toString());
+      out.format(" MAX:%s", meta.getStatistics().genericGetMax().toString());
```

Comment:
Can we format the statistics similar to the values ? https://github.com/apache/parquet-mr/blob/e54ca615f213f5db6d34d9163c97eec98920d7a7/parquet-tools/src/main/java/org/apache/parquet/tools/command/DumpCommand.java#L308
`toString()` is inconsistent since all values are converted to UTF8 strings. Values such as timestamps (INT96) and decimals could be converted to unrecognizable characters like
ST:min:
����
}%, max: ���
}%, num_nulls: 0